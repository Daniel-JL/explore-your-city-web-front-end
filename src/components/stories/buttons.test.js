import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import userEvent from '@testing-library/user-event';
import {
  Router,
} from 'react-router-dom';
import { createMemoryHistory } from 'history';
import { 
  RectButton,
  RoundButton,
  TrashButton,
  RectSubmitButton,
  RoundBackButton,
  RoundAddLocationButton,
  RoundHelpButton,
  TellYourStoryButton,
} from './buttons';

describe('Buttons', () => {
  it('should render correctly', () => {
    const wrapper = renderer.create(<TellYourStoryButton />).toJSON();

    expect(wrapper).toMatchSnapshot();
  });

  it('should do something', () => {

  });
});
