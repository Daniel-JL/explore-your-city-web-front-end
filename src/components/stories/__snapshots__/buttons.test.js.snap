// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons should render correctly 1`] = `
<button
  className="sc-bdfBwQ sc-iBPRYJ dDwjPX RzFWu"
/>
`;
